salt/metalk8s/kubeadm/init/kubelet-start/files/kubeadm.env

```diff
@@ -0,0 +1 @@
+KUBELET_KUBEADM_ARGS={% for key, value in options.items() %}--{{ key }}={{ value }} {% endfor %}
```

Comment:
simple question: does `value` need to be escaped for a shell (as I see `--` like it's forwarded as command line at one point)?

Reply:
Yes it's args for kubelet service start, should maybe at least quote `value`. I make it create a file like kubeadm does